src/scripts/main.js

```diff
@@ -355,6 +355,48 @@ const people = [
 ];
 
 // eslint-disable-next-line no-console
-console.log(people); // you can remove it
 
 // write your code here
+
+// Array of people with detailed information
+const calculateCentury = (year) => {
+  return Math.ceil(year / 100);
+};
+
+const dashboard = document.getElementsByClassName('dashboard')[0];
+
+people.forEach((el) => {
+  const tr = document.createElement('tr');
+
+  const nameCell = document.createElement('td');
+
+  nameCell.textContent = el.name;
+  tr.appendChild(nameCell);
+
+  const genderCell = document.createElement('td');
+
+  genderCell.textContent = el.sex === 'm' ? 'Male' : 'Female';
```

Comment:
The gender assignment logic is correct, but ensure that the data source consistently uses 'm' and 'f' for male and female. If there's any chance of variation, consider normalizing the input data or handling unexpected values.